common/src/main/java/com/wynntils/features/user/overlays/MobTotemTrackingFeature.java

```diff
@@ -105,10 +108,35 @@ public void renderPreview(
 
         void updateRenderTaskCache() {
             renderTaskCache = Models.MobTotem.getMobTotems().stream()
-                    .map(mobTotem -> new TextRenderTask(
-                            "Mob Totem (" + mobTotem.getOwner() + ") at " + mobTotem.getLocation() + " ("
-                                    + mobTotem.getTimerString() + ")",
-                            textRenderSetting))
+                    .map(mobTotem -> {
```

Comment:
Oooh! That is brilliant! Maybe you can extract this into a more general place, so it can be used for shaman totems as well? (It is okay to do that as a follow up)